lib/sync.js

```diff
@@ -102,12 +115,22 @@ async function syncSubmittedFlows(isSudo) {
       await updateParliamentIds(parliamentIdChanges);
     }
 
-    const flows = docs.map(doc => { return { parliamentId: `${doc.pobj}` } });
+    const submittedFlows = docs
+      .filter((doc) => doc.status === VP_PARLIAMENT_FLOW_STATUSES.SUBMITTED)
+      .map(doc => { return { parliamentId: `${doc.pobj}` } });
     await updateParliamentFlowStatus(
-      flows,
+      submittedFlows,
       PARLIAMENT_FLOW_STATUSES.BEING_HANDLED,
       isSudo
     );
+    const refusedFlows = docs
+      .filter((doc) => doc.status === VP_PARLIAMENT_FLOW_STATUSES.REFUSED)
+      .map(doc => { return { parliamentId: `${doc.pobj}` } });
+    await updateParliamentFlowStatus(
+      refusedFlows,
+      PARLIAMENT_FLOW_STATUSES.REFUSED,
+      isSudo
+    );
```

Comment:
Are those the only 2 statuses we are going to get from that API call?
Could we just filter out all REFUSED and handle those separately from the "not" REFUSED?

Reply:
No, there are multiple statuses possible